openedx/features/philu_courseware/views.py

```diff
@@ -51,13 +52,12 @@ def record_and_fetch_competency_assessment(request):
         score:Integer, 1
     :return: JSON
     """
-    competency_record = request.POST.dict()
-    competency_record['user'] = request.user.id
+    competency_record = request.data
```

Comment:
you can show the intent in name that it accepts multiple records by renaming this variable with `competency_records`